schedule/yam/create_hdd_module_combinations_sles12sp5_default_pattern.yaml

```diff
@@ -0,0 +1,19 @@
+name: create_hdd_module_combinations_sles12sp5_default_pattern
```

Comment:
I think the yaml file is same for all qcow for module combination update, so we can name it as a generic one, such as: create_support_image_module_combinations_updated WDYT?

Reply:
OK